starter_code/public/javascript/APIHandler.js

```diff
@@ -1,25 +1,105 @@
+const characters = axios.create({
+  baseURL: "http://localhost:8000/"
+});
+
 class APIHandler {
-  constructor (baseUrl) {
+  constructor(baseUrl) {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
-
+  getFullList() {
```

Comment:
You need to return this promise, otherwise this function will not work.

`
return characters.get("characters/") ... ... ...
`